src/app/event-card/event-card.component.ts

```diff
@@ -117,6 +120,9 @@ export class EventCardComponent implements OnInit {
   isDispatcher() {
     return remult.isAllowed(Roles.dispatcher)
   }
+  isVolunteer() {
+    return remult.authenticated() // TODO How do you identify a volunteer+? Anyone that's authenticated? don't see role
```

Comment:
vounteer vs what? even a manager can be a volunteer

Reply:
Someone who is not a volunteer.

The question, rephrased, is, "does remult.authenticated() guarantee that whoever it may be they're allowed to act like a volunteer?" Presumably there is no issue if a manager starts advertising drives, as they are a manager.